scripts/scr_poststage.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+#
+# SCR allows you to spawn off dataset transfers "in the background"
+# that will finish some time after a job completes.  This saves you from
+# using your compute time to transfer datasets.  You can do this by
+# specifying XFER=BBAPI_POSTSTAGE for a storage descriptor in scr.conf.
+# Currently, this is only supported on on IBM burst buffer nodes.
+#
+# This script is to be run as a 2nd-half post-stage script on an IBM
+# system.  A 2nd-half post-stage script will run after all the job's burst
+# buffers transfers have finished (which could be hours later after the job
+# finishes).  This script will finalize any completed burst buffer dataset
+# transfers so that they're visible to SCR.
+#
+
+# Please fill in BINDIR AND PREFIX below:
+
+# The full path to the dir containing the SCR binaries (scr_index and scr_flush_file)
+BINDIR=""
+
+# The dir path to your SCR PREFIX
+PREFIX="/p/gpfs1/`whoami`"
+
+# Path to where you want the scr_poststage.sh log
+LOGFILE=$PREFIX/scr_poststage.log
+
+if [ ! -d "$BINDIR" ] ; then
+	echo "Please specify BINDIR in scr_poststage.sh" >> $LOGFILE
+	exit
+fi
+
+if [ ! -d "$PREFIX" ] ; then
+	echo "Please specify PREFIX in scr_poststage.sh" >> $LOGFILE
+	exit
+fi
+
+export PATH="$PATH:$BINDIR"
+
+function do_poststage {
+	echo "$(date '+%m/%d/%y %H:%m:%S') Begin post_stage" >> $LOGFILE
+	echo "Current index before finalizing:" >> $LOGFILE
+	scr_index -l -p $PREFIX >> $LOGFILE
+
+	# Finalize each dataset listed in the flush file
+	for ID in $(scr_flush_file --dir $PREFIX --list-ckpt) ; do
```

Comment:
Oh, I just realized there is a ``--list-ckpt`` here.  That does ensure this loop will only operate on checkpoint datasets, so the ``scr_index --current`` would be good.

However, we also want to finalize any pure output (non-checkpoint) datasets.  Refer to the ``scr_postrun`` script, which has a similar requirement but for scavenge operations.